pkg/wireguard/bind.go

```diff
@@ -66,8 +55,7 @@ func (b *IceBind) Close() error {
 		ep.agent.Close()
 	}
 	clear(b.peers)
-	//b.ctxCancel()
-	//b.ctx, b.ctxCancel = context.WithCancel(b.ctx)
+	b.cancel()
```

Comment:
I tried doing this at first and the issue is wireguard-go Closes the bind and re-opens during init so nothing will work because cancelled ctx can't be restarted.